cmd/build/main.go

```diff
@@ -22,7 +22,7 @@ var mgmakeGen []byte
 func main() {
 	ctx := logr.NewContext(context.Background(), mglogr.New("mage-tools-build"))
 	logr.FromContextOrDiscard(ctx).Info("building binary and generating Makefiles...")
-	mageDir := mgpath.FromGitRoot(mgpath.MageDir)
+	mageDir := mgpath.FromMageRoot()
```

Comment:
What's the mage root? The same root as the git root?

Reply:
It's the root dir where all the mage related files are kept - currently hard-coded to `.mage`